code/modules/station_goals/bluespace_tap.dm

```diff
@@ -296,18 +285,18 @@
 			underlays += emissive_appearance(icon, "light_mask")
 
 /obj/machinery/power/bluespace_tap/proc/get_icon_state_number()
-	switch(input_level)
+	switch(mining_power)
 		if(0)
 			return 0
-		if(1 to 3)
+		if(50 KW to 3 MW)
```

Comment:
I think you'd want to extend this range for 0, or just make it the else condition.